zerver/webhooks/sonarqube/doc.md

```diff
@@ -0,0 +1,15 @@
+Get Zulip notifications for your Sonarqube code analysis!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+   By default, the integration will use the project's name as the topic.
```

Comment:
I honestly do not think we need this line about the topic, so would recommend just removing it.

Reply:
Sure